### ISSUE TITLE
supportCircular flag producing error

### ISSUE THREAD

**Issue by VictorOnwukwe** [championswimmer/vuex-persist#164]
When I include the supportCircular flag, despite installing flatted, it produces this error:

JSON.parse(...).map is not a function
    at Module.parse (index.js?5bc5:24)
    at VuexPersistence.restoreState (index.ts?0e3e:202)
    at VuexPersistence.plugin (index.ts?0e3e:238)
    at eval (vuex.esm.js?2f62:352)
    at Array.forEach (<anonymous>)
    at new Store (vuex.esm.js?2f62:352)
    at eval (store.js?c0d6:23)
    at Module../src/store.js (app.js:20774)
    at __webpack_require__ (app.js:724)
    at fn (app.js:101)

**Comment by ajesses** [comment]
hi, I have the same problem. Have you solved it?

**Comment by VictorOnwukwe** [comment]
Nah. I resorted to persisting the storage directly without the use of the plugin...

**Comment by VictorOnwukwe** [comment]
> hi, I have the same problem. Have you solved it?

Nah. I resorted to persisting the storage directly without the use of the plugin...

**Comment by frederikheld** [comment]
Same issue here. I don't really need the feature right now, I just came across the issue when I fiddled around with the different config options if vuex-persists while looking for a fix for a different problem...

**Comment by rogerriche** [comment]
hrrm, this still appears to be a problem.  Has anyone solved this?
